WindowsServerDocs/identity/ad-ds/manage/troubleshoot/Troubleshooting-Active-Directory-Replication-Problems.md

```diff
@@ -20,7 +20,7 @@ The rest of this topic explains tools and a general methodology to fix Active Di
 
 Inbound or outbound replication failure causes Active Directory objects that represent the replication topology, replication schedule, domain controllers, users, computers, passwords, security groups, group memberships, and Group Policy to be inconsistent between domain controllers. Directory inconsistency and replication failure cause either operational failures or inconsistent results, depending on the domain controller that is contacted for the operation, and can prevent the application of Group Policy and access control permissions. Active Directory Domain Services (AD DS) depends on network connectivity, name resolution, authentication and authorization, the directory database, the replication topology, and the replication engine. When the root cause of a replication problem is not immediately obvious, determining the cause among the many possible causes requires systematic elimination of probable causes.
 
-For a UI-based tool to help monitor replication and diagnose errors, see [Active Directory Replication Status Tool](https://www.microsoft.com/download/details.aspx?id=30005)
+For a UI-based tool to help monitor replication and diagnose errors, download and run the [Microsoft Support and Recovery Assistant tool](https://aka.ms/sara-adreplication) OR see [Active Directory Replication Status Tool](https://www.microsoft.com/download/details.aspx?id=30005)

```

Comment:
I acknowledge your wish to emphasize the alternative of the secondary option, but I think we should rather use **bold text** instead, as well as adding a grammatically correct comma:
```suggestion
For a UI-based tool to help monitor replication and diagnose errors, download and run the [Microsoft Support and Recovery Assistant tool](https://aka.ms/sara-adreplication), **or** use the [Active Directory Replication Status Tool](https://www.microsoft.com/download/details.aspx?id=30005) if you only want to analyze the replication status.
```
(This phrasing might be insufficient, so feel free to suggest what you see as the most correct explanation for the old tool.)